From 2d5a51143d224e04891860d12f6bd91864fda001 Mon Sep 17 00:00:00 2001
From: Vladislav Vaintroub <wlad@mariadb.com>
Date: Sat, 8 May 2021 20:23:50 +0200
Subject: [PATCH] Windows - fix typo, that makes clang-cl compilation fail.

It is _WIN32 that is always defined on Windows, not anything else.
---
 include/ma_crypt.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/ma_crypt.h b/include/ma_crypt.h
index d46bb0e2d..cf73359e7 100644
--- a/include/ma_crypt.h
+++ b/include/ma_crypt.h
@@ -44,7 +44,7 @@
 #define MA_MAX_HASH_SIZE 64
 /** \typedef MRL hash context */
 
-#if defined(WIN32)
+#if defined(_WIN32)
 #include <windows.h>
 #include <bcrypt.h>
 typedef struct {
@@ -153,7 +153,7 @@ static inline void ma_hash(unsigned int algorithm,
                            unsigned char *digest)
 {
   MA_HASH_CTX *ctx= NULL;
-#ifdef HAVE_SCHANNEL
+#ifdef _WIN32
   MA_HASH_CTX dctx;
   ctx= &dctx;
 #endif
